paperify.sh

```diff
@@ -61,11 +61,11 @@ do
   out="${f/\ /_}.png"
   cat "$f" | qrencode --8bit -v 40 --size=13 --margin=1 --output "$out"
   convert -size 2490x3510 xc:white \( $out -gravity center \) -composite \
-    -font $FONT -pointsize 72 -gravity northwest -annotate +100+200 "FILE: $filename\n\nCHUNK: $f\n\nTOTAL CHUNKS: $count" \
```

Comment:
Is it possible if we move these font changes to another PR so we can discuss them separately?